deploy/helm/minikube-dev.yml

```diff
@@ -13,9 +13,6 @@ global:
   enablePriorityClass: false
   defaultContactEmail: "magda@mailinator.com"
   enableMultiTenants: false
-  openfaas:
-    # turn off auth over openfass gateway for ease of debugging
-    allowAdminOnly: false
```

Comment:
This values is actually required by gateway (when it forward request to openfaas). 
We also have some other optional setting under `global.openfaas`:
- namespacePrefix: "" # auto determined from the actual Magda-core deployment namespace
- functionNamespace: openfaas-fn  # Default namespace for functions
- mainNamespace: openfaas # Default namespace for gateway and other core modules
Thought this one is more relevant to openfaas. Thus, put here.